### ISSUE TITLE
inlets-operator read license from file / create k8s secret for license

### ISSUE THREAD

**Issue by Waterdrips** [alexellis/arkade#102]
once support for the license being a secret is merged into the inlets-operator chart it would be good to support that here, so the license is not exposed in deployments etc.

## Expected Behaviour

allow a flag of --license-file (or similar) to create the k8s secret and pass it to the chart (once this issue is implemented: https://github.com/inlets/inlets-operator/issues/67 )

## Current Behaviour

passed as a literal

## Possible Solution

see how the token file is done, and mimic that behavior

**Comment by alexellis** [comment]
Sounds good to me

**Comment by alexellis** [comment]
@Waterdrips 👋  what's the status of this one please?
